src/extension.ts

```diff
@@ -410,6 +411,24 @@ class DeviceTreeItem extends vscode.TreeItem {
         }
     }
 
+    public async printSysinfo() {
+        const messageBarItem = new StatusBarProgressionMessage("Grabbing ev3dev system info...");
+        try {
+            const sysinfo = await this.device.getSystemInfo();
+
+            output.appendLine("========== ev3dev-sysinfo ==========");
+            output.appendLine(sysinfo);
+            output.appendLine("");
+            output.show();
+
+            messageBarItem.finish("System info downloaded; see output window.");

```

Comment:
This is a bit long. We can just say 'System info retrieved'. The output window will be shown, so it will have already grabbed the users attention.

As a side note, I have been following the LEGO convention that downloading means host > EV3 and upload means EV3 > host.